cg/apps/cgstats/stats.py

```diff
@@ -78,7 +92,7 @@ def is_lane_pooled(self, flowcell_obj: models.Flowcell, lane: str) -> bool:
 
     def sample_reads(self, sample_obj: models.Sample) -> Iterator:
```

Comment:
think this typehint is wrong and it returns a Query object. But it was there before you, so

Reply:
perhaps you can make a suggestion for a better type hint?

Reply:
alchy.Query

Reply:
changed